### ISSUE TITLE
Significant overfit on default hyperparameters on cornell-movie-dialogs config

### ISSUE THREAD

**Issue by lk251** [DongjunLee/conversation-tensorflow#11]
Running:
python main.py --config cornell-movie-dialogs --mode train

to the end (100000 steps) will result in a training loss of about 2.6, test loss of 8.4.

Which hyperparameters did you use? The resulting chatbot doesn't work very well (the one in your readme is a lot better).

Thank you!

**Comment by DongjunLee** [comment]
Seq2Seq model has a problem that there is a gap between training and inference.
You can check practical tips for training sequence-to-sequence models with attention in [this blog](https://awni.github.io/train-sequence-models/?utm_campaign=Revue%2520newsletter&utm_medium=Newsletter&utm_source=The%2520Wild%2520Week%2520in%2520AI)

I don't remember the model's detail because I worked on it before.
I guess, The result in readme is more overfitting than yours. (200,000 steps)

And cornell-movie-dialogs is too small to train conversation model. 
Lack of data means that overfitting is inevitable.

**Comment by lk251** [comment]
Thanks for the advice!
